### PR TITLE
Add options kured

### PR DESCRIPTION
**Pull request by captnCC**
This MR adds the ability to configure kured with additional options (and override defaults)

### CODE REVIEW

File: templates/kured.yaml.tpl [hunk 1]
```diff
@@ -21,3 +21,7 @@ spec:
             - --pre-reboot-node-labels=kured=rebooting
```
Comment (mysticaltech) [inline templates/kured.yaml.tpl:21]: @captnCC Those actually get re-included below with your new logic. So they need to get deleted, no?

**Review by mysticaltech — CHANGES_REQUESTED** [review]
Looks great, just see the tiny comment about previous kured arguments, those may need to be removed.

File: templates/kured.yaml.tpl (hunk 1 above)
Comment (captnCC) [reply templates/kured.yaml.tpl]: Yes that was an oversight, they are gone now

File: variables.tf [hunk 2]
```diff
@@ -390,6 +390,11 @@ variable "rancher_values" {
   description = "Additional helm values file to pass to Rancher as 'valuesContent' at the HelmChart."
 }
 
+variable "kured_options" {
+  type    = map(string)
+  default = {}
```
Comment (mysticaltech) [inline variables.tf:395]: @captnCC Now we need to re-add the defaults in here.

Comment (captnCC) [reply variables.tf:395]: @mysticaltech They will be actually merged with the defaults in the `locals.tf`

Comment (mysticaltech) [reply variables.tf:395]: My bad, indeed! :)

**Review by mysticaltech — APPROVED** [review]

**Comment by mysticaltech** [comment]
@captnCC If you've done a test deployment, feel free to merge to master, and will do a quick release! As this adds a small new feature, I believe we can move to 1.8, it's about time lol.

Otherwise, if you want me to try it too, you can merge it to staging, and will take care of it ASAP.

**Comment by mysticaltech** [comment]
@captnCC Released, thanks! 🙏🚀

**Pull request merged**